gctscript/vm/vm_task.go

```diff
@@ -0,0 +1,30 @@
+package vm
+
+import (
+	"time"
+
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+func (vm *VM) runner() {
+	vm.S = make(chan struct{}, 1)
+	waitTime := time.NewTicker(vm.T)
+	vm.NextRun = time.Now().Add(vm.T)
+
+	go func() {
+		for {
+			select {
+			case <-waitTime.C:
+				vm.NextRun = time.Now().Add(vm.T)
```

Comment:
As of b1247d627a4b794174c47bfb47f4c505bda2bc41 this doesn't appear to be hit via tests on my system